src/nbl/video/CVulkanPhysicalDevice.h

```diff
@@ -62,7 +61,7 @@ class CVulkanPhysicalDevice final : public IPhysicalDevice
         // Get physical device's features
         {
             VkPhysicalDeviceFeatures features;
-            vkGetPhysicalDeviceFeatures(m_vkphysdev, &features);
+            vkGetPhysicalDeviceFeatures(m_vkPhysicalDevice, &features);
                     
```

Comment:
are there any more features we can expose?

Reply:
Just did the ray tracing ones for now as they're relevant to Erfan's work. I suggest to expose more as we need them.